tests/Utilities/TokenGenerator.php

```diff
@@ -37,12 +37,12 @@ protected static function getIdTokenClaims(
         $defaults = [
             'sub' => '__test_sub__',
             'iss' => 'https://__test_domain__/',
-            'aud' => '__test_client_id__',
+            'aud' => 'https://__test_domain__/userinfo',
```

Comment:
As per the issue, the Client ID is never used for ID tokens. However, all ID tokens contain `$DOMAIN/userinfo` and `$applicationName` in `aud`.